### DISCUSSION TITLE
Unable to use https proxy for a https URL

### DISCUSSION THREAD

**Discussion by rghv404** [encode/httpx#2918] [Potential Issue]
I'm using scrapingbee proxy as :

```
proxies = {
    'http://': f'http://{SCRAPINGBEE_API_KEY}@proxy.scrapingbee.com:8886',
    'https://': f'https://{SCRAPINGBEE_API_KEY}@proxy.scrapingbee.com:8887'}

```
However I get below error:
`RuntimeError: Attempted to add a TLS layer on top of the existing TLS stream, which is not supported by httpcore package`
 

As far as I understand, https proxies were enabled in httpcore package here: https://github.com/encode/httpcore/issues/722. 
What could be causing this?

P.S Using requests library with the above proxy works just fine:
```
res = requests.get(
    url=MY_URL,
    proxies=proxies,
)
```

**Comment by zanieb** [comment]
What versions are you using?

```
pip show httpx httpcore
```

**Reply by rghv404** [reply]
The latest one: 0.18.0

```
Name: httpx
Version: 0.25.0
Summary: The next generation HTTP client.
Home-page: 
Author: 
Author-email: Tom Christie <tom@tomchristie.com>
License: 
Requires: certifi, httpcore, idna, sniffio
Required-by: 
---
Name: httpcore
Version: 0.18.0
Summary: A minimal low-level HTTP client.
Home-page: 
Author: 
Author-email: Tom Christie <tom@tomchristie.com>
License: 
Requires: anyio, certifi, h11, sniffio
Required-by: httpx
```

**Comment by karpetrosyan** [comment]
Closing due to a duplicate of https://github.com/encode/httpcore/discussions/833
